code/game/objects/items/tools/wirecutters.dm

```diff
@@ -92,4 +95,16 @@
 	var/obj/item/crowbar/power/pryjaws = new /obj/item/crowbar/power
 	to_chat(user, "<span class='notice'>You attach the pry jaws to [src].</span>")
 	qdel(src)
-	user.put_in_active_hand(pryjaws)
\ No newline at end of file
+	user.put_in_active_hand(pryjaws)
+	
+/obj/item/wirecutters/power/attack(mob/living/carbon/C, mob/user)
+	if(istype(C) && C.handcuffed)
+		user.visible_message("<span class='notice'>[user] cuts [C]'s restraints with [src]!</span>")
+		qdel(C.handcuffed)

```

Comment:
When #33821 is merged, simplify this to just the qdel

Reply:
Will do it now, as I wanted to wait until your PR was done anyway